src/components/Room/RoomSlider.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import Slider from '@material-ui/core/Slider';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: 280
+  }
+}));
+const CustomSlider = withStyles({
+  root: {
+    color: '#96a0fa',
+    height: 2
+  },
+  thumb: {
+    height: 25,
+    width: 25,
+    backgroundColor: '#1f2666',
+    marginTop: -10,
+    marginLeft: -12,
+    '&:focus,&:hover,&$active': {
+      boxShadow: 'inherit'
+    }
+  },
+  active: {},
+  track: {
+    height: 5,
+    borderRadius: 3
+  },
+  rail: {
+    height: 5,
+    borderRadius: 3
+  }
+})(Slider);
+
+function RoomSlider(props) {
+  const classes = useStyles();
+  const [, setValue] = React.useState(20);
+  const handleChange = (event, newValue) => {
```

Comment:
Throws error: setValue not defined. I don't get why it would be necessary to memoize the results of this function? Would it improve performance for such a simple function